ha/fault_tolerance/fault_tolerance_driver.py

```diff
@@ -21,78 +21,34 @@
 
 
 import time
-import traceback
 
-from cortx.utils.conf_store import Conf
 from cortx.utils.log import Log
-from ha.util.message_bus import MessageBus, CONSUMER_STATUS, MessageBusConsumer
 
-from ha import const
 from ha.core.config.config_manager import ConfigManager
-from ha.core.event_analyzer.event_analyzerd import EventAnalyzer
-from ha.k8s_setup.const import _DELIM
-
-from consul.base import ConsulException
-from cortx.utils.conf_store.error import ConfError
-from ha.core.event_analyzer.event_analyzer_exceptions import EventFilterException
-from ha.core.event_analyzer.event_analyzer_exceptions import EventParserException
-from ha.core.event_analyzer.event_analyzer_exceptions import SubscriberException
+from ha.fault_tolerance.fault_monitor import NodeFaultMonitor
+from ha.fault_tolerance.cluster_stop_monitor import ClusterStopMonitor
 
 class FaultTolerance:
     """
     Module responsible for consuming messages from message bus,
     further analyzes that event and publishes it if required
     """
+    #def __init__(self, wait_time=600):
```

Comment:
Please remove this commented code if not required.

Reply:
Done